content/product/product_management/okrs.md

```diff
@@ -0,0 +1,58 @@
+# Product and engineering OKRs
+
+## Philosophy
+
+### Product and engineering share OKRs
+
+Product and engineering have a shared goal to deliver a product that provides value to our customers, so we set shared OKRs.
+
+### OKRs are a contract
+
+OKRs are a contract between different management levels at our company.
+
+```
+CEO <-> Company OKRs <-> Executive team
+Product/Engineering VPs <-> Product/Engineering OKRs <-> Product/Engineering Directors
+Product/Engineering Directors <-> Product/Engineering org OKRs <-> Product/Engineering teams
+```
+
+### OKRs do not nest or cascade
```

Comment:
It doesn't exactly fit in this section, but doesn't fit in the others either.. maybe we should have something about not setting too many OKRs so we don't dilute focus?